app/helpers/solidus_paypal_commerce_platform/venmo_standalone_options_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module SolidusPaypalCommercePlatform
+  module VenmoStandaloneOptionsHelper
+    def preferred_venmo_standalone_options
```

Comment:
The other helper was used for the PayPal button style and is named `ButtonOptionsHelper`, I thought it would more sense that it is under another name so it is not confusing

Reply:
👍